dev_guide/expanding_persistent_volumes.adoc

```diff
@@ -15,8 +15,23 @@ toc::[]
 
 To allow expansion of persistent volume claims (PVC) by {product-title} users,
 {product-title} administrators must create or update a StorageClass with
-`allowVolumeExpansion` set to `true`. Only PVCs created from that class are
-allowed to expand.
+`allowVolumeExpansion` set to `true`. Only PVCs created from that class are allowed to expand.
+
+For example, {product-title} administrators add the `allowVolumeExpansion` attribute at the bottom of the StorageClass’s configuration:
```

Comment:
It feels like a word is missing here. 
_{product-title} administrators can add... _ ??

Reply:
I'll update and recommit with your and Kathryn's suggestions.